resdk/resolwe.py

```diff
@@ -70,7 +71,7 @@ def _version_string_to_int(self, version):
         version_numbers = [int(number_string) for number_string in version.split(".")]
```

Comment:
I did not want a dependency to anything related to Django in the SDK. The django-versionfield does not define a Django requirement in the `setup.py`, but it should. It imports modules from Django. I suggest to make the SDK as lean as possible.